ros_ws/src/crazyswarm/scripts/test_collisionAvoidance.py

```diff
@@ -42,9 +44,123 @@ def test_velocityMode_sidestepWorstCase(args=None):
         timeHelper.sleep(timeHelper.dt)
         if a.position()[0] > 1.0 and b.position()[0] < -1.0:
             return
-
     assert False
 
 
+def test_goToWithoutCA_CheckCollision():
+    args = "--sim --vis null"
+    allcfs, timeHelper = setUp(args)
+    allcfs.takeoff(targetHeight=Z, duration=1.0+Z)
+    timeHelper.sleep(1.5+Z)
+    cf0, cf1 = allcfs.crazyflies
+
+    cf0.goTo([1.0, 0.0, 1.0], 0, 5.0)
+    cf1.goTo([-1.0, 0.0, 1.0], 0, 5.0)
+    # flag if a collision happened during sleep
+    collisionHappend = False
+    while timeHelper.time() < 10.0:
+        positions = np.stack([cf0.position(), cf1.position()])
+        collisions = check_ellipsoid_collisions(positions, RADII)
+        if np.any(collisions):
+            collisionHappend = True
+
+        timeHelper.sleep(timeHelper.dt)
+        if cf0.position()[0] > 1.0 and cf1.position()[0] < -1.0:
+            return
+    assert collisionHappend
+
+
+def test_goToWithCA_CheckCollision():
+    args = "--sim --vis null"
+    allcfs, timeHelper = setUp(args)
+    allcfs.takeoff(targetHeight=Z, duration=1.0+Z)
+    timeHelper.sleep(1.5+Z)
+    cf0, cf1 = allcfs.crazyflies
+    cf0.enableCollisionAvoidance([cf1], RADII)
+    cf1.enableCollisionAvoidance([cf0], RADII)
+
+    cf0.goTo([1.0, 0.0, 1.0], 0, 5.0)
+    cf1.goTo([-1.0, 0.0, 1.0], 0, 5.0)
+    while timeHelper.time() < 10.0:
+        positions = np.stack([cf0.position(), cf1.position()])
+        collisions = check_ellipsoid_collisions(positions, RADII)
+        assert not np.any(collisions)
+
+        timeHelper.sleep(timeHelper.dt)
+        if cf0.position()[0] > 1.0 and cf1.position()[0] < -1.0:
```

Comment:
this check is not robust to floating point errors. it should be something like `cf0.position()[0] > 1.0 - 1e-6`. Same for the previous test